### PR TITLE
feat: add basic service and installer

### PR DESCRIPTION
**Pull request by KwadFan**
This adds a basic installer and a basic service

Signed-off-by: Stephan Wendel <me@stephanwe.de>

### CODE REVIEW

**Review by roamingthings — APPROVED** [review]
Not being the expert shell guy, this looks good to me

File: scripts/spyglass [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+### Helper Messages
+debug_msg() {
+    printf "DEBUG: %s\n" "${1}"
+}
+
+help_msg() {
+    echo -e "spyglass - Picamera2 MJPG Streamer\nUsage:"
+    echo -e "\t crowsnest [Options]"
+    echo -e "\n\t\t-h Prints this help."
+    echo -e "\n\t\t-c </path/to/configfile>\n\t\t\tPath to your webcam.conf\n"
```
Comment (mryel00) [inline scripts/spyglass:29]: Some copy-paste error?

**Review by mryel00 — COMMENTED** [review]
Looks good except for one copy-paste error

**Comment by KwadFan** [comment]
> Looks good except for maybe one copy-paste error

Show me which one, I'll fix it asap.

**Review by mryel00 — APPROVED** [review]

**Comment by roamingthings** [comment]
I've added a feature request regarding versioning #19

**Pull request merged**